docs/builds/guides/integration/installation.md

```diff
@@ -44,6 +45,10 @@ npm install --save @ckeditor/ckeditor5-build-decoupled-document
 
 CKEditor will then be available at `node_modules/@ckeditor/ckeditor5-build-[name]/build/ckeditor.js`. It can also be imported directly to your code by `require( '@ckeditor/ckeditor5-build-[name]' )`.
 
+### Online builder
+
+Go to the [Online builder page](https://ckeditor.com/ckeditor-5/online-builder/), go through a few creator steps to pick plugins and toolbar items and visualize the editor. Then download the prepared build with a simple instruction how to run it.
```

Comment:
The [online builder](https://ckeditor.com/ckeditor-5/online-builder/) lets you download CKEditor 5 builds and also allows you to create your own, customized builds (with a different set of plugins) in a few easy steps, through a simple and intuitive UI.